discord/utils.py

```diff
@@ -396,11 +392,11 @@ def find(predicate: Callable[[T], Any], iterable: Iterable[T], /) -> Optional[T]
 
 
 @overload
-def find(predicate: Callable[[T], Any], iterable: AsyncIterable[T], /) -> Coro[Optional[T]]:
+def find(predicate: Callable[[T], Any], iterable: AsyncIterable[T], /) -> Awaitable[Optional[T]]:
```

Comment:
Surely this is better off using Coro so the return type is more specific?

Reply:
Good call. I'll revert get() as well.